### DISCUSSION TITLE
Support GitHub markdown as a BYO content

### DISCUSSION THREAD

**Discussion by TsayAdobe** [adobe/helix-home#303] [Ideas]
There are some advantages of using the markdown format. 

1. It is easier to diff the content changes.
2. The existing markdown content can be moved to Franklin directly.
3. Content can be edited in a simple text editor.
4. Content is more portable than Google or Word Docs.
5. Content automation is easer in the markdown format

Users use a GitHub repo as the content repo. The Chrome plugin can be enabled when an author edits a markdown in GitHub.com. A VS Code plugin can be provided if the editor is used for authoring content.

**Comment by tripodsan** [comment]
> 1. It is easier to diff the content changes.

I think the track-changes feature of word/gdocs is very powerful and allow for annotation. However, I think the commit/PR feature of git allow for a more trackable workflow.

**Comment by tripodsan** [comment]
> 2. The existing markdown content can be moved to Franklin directly.

I don't think that this is true. there needs to be some validation. also, not all feature that the helix markdown is using (like gridtables) can be edited easily with the standard markdown tools

**Comment by tripodsan** [comment]
> 3. Content can be edited in a simple text editor.

true. but this is mostly a problem for many authors :-)

**Comment by tripodsan** [comment]
>  4. Content is more portable than Google or Word Docs.

portable to what? docx is a very common format and many other document-processing tools allow to import it. less so with markdown.

**Comment by tripodsan** [comment]
> 5. Content automation is easer in the markdown format

depends on the use case. having content in gdrive/sharepoint allow for automation if google scripts or power automate tools.

**Comment by tripodsan** [comment]
> Users use a GitHub repo as the content repo. The Chrome plugin can be enabled when an author edits a markdown in GitHub.com. A VS Code plugin can be provided if the editor is used for authoring content.

we allowed to use md content from github in former helix versions but removed it since nobody used it. but if there is a customer that wants to use it, we can turn it on again.
